cli/src/main/java/com/devonfw/tools/ide/tool/jmc/Jmc.java

```diff
@@ -33,7 +32,7 @@ public Jmc(IdeContext context) {
   @Override
   public boolean doInstall(boolean silent) {
 
-    getCommandlet(Java.class).install();
+    // getCommandlet(Java.class).install();
```

Comment:
I would probably add here a todo comment, so that it won't get lost if we have [#131](https://github.com/devonfw/IDEasy/issues/131) implemented. Please also add a todo comment in the jmc test concerning solving the binding port problem. Thank you!

Reply:
This change is for #131 and does not belong into this PR. We should revert this.